backstory/src/test/java/com/google/sps/story/StoryEndingToolsTest.java

```diff
@@ -0,0 +1,229 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.story;
+
+import com.google.sps.story.data.StoryEndingTools;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Quality Tests for StoryEndingTools
+ */
+@RunWith(JUnit4.class)
+public final class StoryEndingToolsTest {
+
+  // strings to use in testing story ending tools
+  private final static String EMPTY_TEXT = "";
+  private final static String SENTENCE_FRAGMENT = "This story is not ";
+  private final static String COMPLETE_SENTENCE = "This story is not incomplete.";
+  private final static String QUESTION_FRAGMENT = "What is this? I don't ";
+  private final static String COMPLETE_QUESTION = "I didn't realize. What's happening here?";
+  private final static String EXCLAMATION_FRAGMENT = "Oh no! This isn't ";
+  private final static String COMPLETE_EXCLAMATION = "I had no idea. Oh no!";
+
+  // aggregate all of these into one input array
+  private final static String[] INPUTS = { EMPTY_TEXT, SENTENCE_FRAGMENT, COMPLETE_SENTENCE, 
+                                    QUESTION_FRAGMENT, COMPLETE_QUESTION, 
+                                    EXCLAMATION_FRAGMENT, COMPLETE_EXCLAMATION };
+
+  // array of sentence enders
+  private final static String[] SENTENCE_ENDERS = {".", "?", "!"};
```

Comment:
Can you make the StoryEndingTools.SENTENCE_ENDERS field public instead? That avoids the need to keep these two in sync.

Reply:
Sure.

Reply:
Done